templates/serverpod_templates/projectname_flutter/android/local.properties

```diff
@@ -0,0 +1,2 @@
+sdk.dir=/Users/apple/Library/Android/sdk
```

Comment:
Avoid committing files that aren't part of the actual feature you are working on, this makes it much harder to review.

Reply:
ok, sorry about that. i didn't check it before pushing it